mkdocs.yml

```
@@ -88,6 +88,7 @@ nav:
       - Implement Sampling Strategies: how-to-guides/sampling.md
       - Export your Logfire Data: how-to-guides/query-api.md
       - Scrub Sensitive Data: how-to-guides/scrubbing.md
+      - Trace across Processes: how-to-guides/distributed-tracing.md
```

```suggestion
      - Distributed Tracing: how-to-guides/distributed-tracing.md
```
Not sure where the "Trace across Processes" name came from, I understand if "Distributed Tracing" seems too jargony, but reading "across Processes" makes me think it's more of a process-pool thing in a single python program, than an across-different-hardware-instances sort of thing.

If you don't like "Distributed Tracing", I would suggest:
```suggestion
      - Tracing across multiple services: how-to-guides/distributed-tracing.md
```

---

all the headings here complete the sentence 'How to'. Not sure I like that but I was going along with it.